### PR TITLE
Implement EntityShootBowEvent

### PR DESCRIPTION
**Pull request by LuminousLoreBloom**

### CODE REVIEW

**Comment by LuminousLoreBloom** [comment]
~~Currently pending, I am yet to make a change in Glowkit to enhance the API.~~

**Comment by LuminousLoreBloom** [comment]
PR ready but depends on https://github.com/GlowstoneMC/Glowkit/pull/19

File: src/main/java/net/glowstone/block/itemtype/ItemBow.java [hunk 1]
```diff
@@ -74,40 +77,58 @@ public void endUse(GlowPlayer player, ItemStack bow) {
...
-            if (player.getInventory().getItemInMainHand().getType() == Material.BOW) {
-                player.getInventory().setItemInMainHand(InventoryUtil.damageItem(player, bow));
+            if (event.isCancelled()) {
+                launchedProjectile.remove();
             } else {
-                player.getInventory().setItemInOffHand(InventoryUtil.damageItem(player, bow));
+                chargeFraction = event.getForce();
+                launchedProjectile = (Projectile) event.getProjectile();
+                if (consumeArrow) {
+                    int amount = arrow.getAmount();
```
Comment (Pr0methean) [inline src/main/java/net/glowstone/block/itemtype/ItemBow.java]: Use InventoryUtil.consumeItem().

**Comment by LuminousLoreBloom** [comment]
How do I update Glowkit? The API version is unchanged? I guess build just needs to rerun.

Away from home for two weeks but I'll do the consumeItem change when I'm back.

**Comment by aramperes** [comment]
It would be better if the Glowkit version was bumped, but a rebuild should work nonetheless.

**Review by Pr0methean — APPROVED** [review]

**Comment by Pr0methean** [comment]
LGTM, but needs an in-game test.

**Pull request merged**